test/ccpm_test.go

```diff
@@ -0,0 +1,152 @@
+		{args: "ccpm plugin version list --plugin ccp-123456 --environment env-123456 -o json", fixture: "ccpm/plugin-version-list-json.golden"},
+		{args: "ccpm plugin version list --plugin ccp-123456 --environment env-123456 -o yaml", fixture: "ccpm/plugin-version-list-yaml.golden"},
+
+		// Version create tests - using exact plugin IDs from test-server
+		{args: "ccpm plugin version create --plugin ccp-123456 --version 1.0.0 --plugin-file test/fixtures/input/connect/confluentinc-kafka-connect-datagen-0.6.1.zip --connector-classes 'io.confluent.kafka.connect.datagen.DatagenConnector:SOURCE' --environment env-123456", fixture: "ccpm/plugin-version-create.golden"},
+		{args: "ccpm plugin version create --plugin ccp-123456 --version 1.0.0 --plugin-file test/fixtures/input/connect/confluentinc-kafka-connect-datagen-0.6.1.zip --connector-classes 'io.confluent.kafka.connect.datagen.DatagenConnector:SOURCE' --sensitive-properties 'password,secret' --documentation-link 'https://docs.confluent.io' --environment env-123456", fixture: "ccpm/plugin-version-create-with-options.golden"},
+		{args: "ccpm plugin version create --plugin ccp-123456 --version 1.0.0 --plugin-file test/fixtures/input/connect/invalid-file.txt --connector-classes 'io.confluent.kafka.connect.datagen.DatagenConnector:SOURCE' --environment env-123456", fixture: "ccpm/plugin-version-create-invalid-file.golden", exitCode: 1},
+
+		// Version describe tests - using exact IDs from test-server
+		{args: "ccpm plugin version describe ver-123456 --plugin ccp-123456 --environment env-123456", fixture: "ccpm/plugin-version-describe.golden"},
+		{args: "ccpm plugin version describe ver-123456 --plugin ccp-123456 --environment env-123456 -o json", fixture: "ccpm/plugin-version-describe-json.golden"},
+		{args: "ccpm plugin version describe ver-123456 --plugin ccp-123456 --environment env-123456 -o yaml", fixture: "ccpm/plugin-version-describe-yaml.golden"},
+		{args: "ccpm plugin version describe invalid-version --plugin ccp-123456 --environment env-123456", fixture: "ccpm/plugin-version-describe-not-found.golden", exitCode: 1},
+
+		// Version delete tests - using exact IDs from test-server
+		{args: "ccpm plugin version delete ver-123456 --plugin ccp-123456 --environment env-123456 --force", fixture: "ccpm/plugin-version-delete.golden"},
+		{args: "ccpm plugin version delete ver-123456 --plugin ccp-123456 --environment env-123456", input: "y\n", fixture: "ccpm/plugin-version-delete-prompt.golden"},
+		{args: "ccpm plugin version delete ver-123456 --plugin ccp-123456 --environment env-123456", input: "n\n", fixture: "ccpm/plugin-version-delete-cancelled.golden"},
+		{args: "ccpm plugin version delete invalid-version --plugin ccp-123456 --environment env-123456 --force", fixture: "ccpm/plugin-version-delete-not-found.golden", exitCode: 1},
+	}
+
+	for _, test := range tests {
+		test.login = "cloud"
+		s.runIntegrationTest(test)
+	}
+}
+
+func (s *CLITestSuite) TestCCPMValidation() {
```

Comment:
```suggestion
func (s *CLITestSuite) TestCCPM_Validation() {
```

Nitpick: Just a style thing. We try to separate any descriptions that aren't part of the command with an underscore